src/cli.ts

```diff
@@ -582,6 +583,8 @@ export function parseArgs(args: yargs.Argv<RawOptions>): RawOptions {
   ]) {
     if (parsed[arg] && typeof parsed[arg] === 'string') {
       parsed[arg] = parseJson(parsed[arg] as string);
+      // also set camel-cased option keys
```

Comment:
Pls replace with a comment that says _why_, not just _what_. The _what_ is the code, and in such a case the code is sufficiently simple to not need a translation to English, but a _why_ giving context will be appreciated by our future selves or other maintainers wondering why we do this 🙂.

Reply:
Please check the latest commit. Have explained a bit more now.